packages/reassure-measure/package.json

```diff
@@ -38,8 +38,13 @@
   "dependencies": {
     "mathjs": "^10.1.1"
   },
+  "devDependencies": {
+    "@types/jest": "^28.0.0",
+    "typescript": "^4.5.2"
```

Comment:
I think we don't need to declare those per package, should be fine to only do it in root package.json?